database/setup.sql

```diff
@@ -0,0 +1,43 @@
+IF db_id('studyzen') IS NULL CREATE DATABASE studyzen;
```

Comment:
Database setup should be in Infrastructure/Persistence. Each domain entity should be configured using EF core. Look into this: https://learn.microsoft.com/en-us/ef/core/modeling/

Also, SQL has type Text, which we should probably use for lecture content. Haven't used it, but I think it's optimized for storing long texts

Reply:
So I understand that we are going to use code-first database, so for now added the initial configurations